src/main/java/com/app/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,24 @@
 package com.app.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.Length;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @Length(min = 1, max = 255)
     private String title;

```

Comment:
I guess, in this case, we should use NotBlank annotation also because someone can enter such title as "  " (two whitespaces)